lib/inspec_tools/inspec.rb

```diff
@@ -220,9 +230,12 @@ def generate_ip
       ip
     end
 
-    def generate_title(title, json, date)
+    # @param attributes XCCDF attributes from source specification
+    def generate_title(attributes, json)
```

Comment:
The README needs to be updated about the behavior change when generating titles based on these changes.